public/js/lib/editor/index.js

```diff
@@ -693,7 +717,7 @@ export default class Editor {
       readOnly: true,
       autoRefresh: true,
       otherCursors: true,
-      placeholder: "← Start by entering a title here\n===\nVisit /features if you don't know what to do.\nHappy hacking :)"
+      placeholder: "Partagez-nous vos retours sur LanguageTool : \nhttps://grist.incubateur.net/o/docs/forms/mxmyvAB6sBThkX5CarmwZt/4 \n\nActiver/désactiver le correction à l'aide de la croix blanche en bas de page ↓"
```

Comment:
Not sure I'd say all this here since we can't select the placeholder text to copy/paste the complex URL. And talking about "languagetool" is pretty dev-centric, the user won't know what you are talking about.

I think I'd just remove this change for now and maybe add something in the /features pad or the help modal.

Reply:
As stated in the commit, the comment was just there for the "private" demonstration in staging, and will be dropped before merging. I noticed the copy/paste was not working too late :( I'll simply drop this comment.

Reply:
(LanguageTool is a brand, as grammarly, or Merci App, it's not reserved for dev)